app/modules/encounters/resources.py

```diff
@@ -174,19 +95,90 @@ def get(self, encounter):
     )
     @api.login_required(oauth_scopes=['encounters:write'])
     @api.parameters(parameters.PatchEncounterDetailsParameters())
-    @api.response(schemas.DetailedEncounterSchema())
     @api.response(code=HTTPStatus.CONFLICT)
     def patch(self, args, encounter):
         """
         Patch Encounter details by ID.
         """
+
+        edm_count = 0
+        for arg in args:
```

Comment:
@karenc is there a more pythonic way to do this using list comprehension?

Reply:
May be like this?

```python
edm_args = [arg for arg in args if arg.get('path') in parameters.PatchEncounterDetailsParameters.PATH_CHOICES_EDM

if edm_args and len(edm_args) != len(args):
```